### PR TITLE
perf(executor): batch commit changes by all txs in one block

### PR DESCRIPTION
**Pull request by wenyuanhust**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Which docs this PR relation**:

Ref #

**Which toolchain this PR adaption**:

No Breaking Change

**Special notes for your reviewer**:

### CI Switch

ci-runs-only: [Chaos CI, Cargo Clippy, Coverage Test, E2E Tests, Code Format, Unit Tests]

### CODE REVIEW

File: core/executor/src/lib.rs [hunk 1]
```diff
@@ -94,14 +94,13 @@ impl Executor for AxonExecutor {
         let mut hashes = Vec::with_capacity(txs_len);
         let mut gas_use = 0u64;
 
-        let evm_executor = EvmExecutor::new();
-        let sys_executor = SystemExecutor::new();
-
```
Comment (KaoImin) [inline core/executor/src/lib.rs]: Why change this?

**Comment by KaoImin** [comment]
Please change the PR type into `perf`

File: core/executor/src/lib.rs (hunk 1 above)
Comment (wenyuanhust) [reply core/executor/src/lib.rs]: a mistake..

File: core/executor/src/lib.rs [hunk 2]
```diff
@@ -111,14 +111,16 @@ impl Executor for AxonExecutor {
             hashes.push(Hasher::digest(&r.ret));
             res.push(r);
         }
+        // commit changes by all txs included in this block only once
+        let state_root = backend.commit();
```
Comment (KaoImin) [inline core/executor/src/lib.rs]:
```suggestion
        let new_state_root = backend.commit();
```

**Review by KaoImin — APPROVED** [review]

**Pull request merged**